src/menu/src/MenuItem.js

```diff
@@ -146,11 +131,6 @@ MenuItem.propTypes = {
    */
   intent: PropTypes.oneOf(['none', 'success', 'warning', 'danger']),
 
-  /**
-   * Callback to invoke onkeypress
-   */
-  onKeyPress: PropTypes.func,
```

Comment:
What about turning this into` onKeyDown`?

Reply:
It doesnt need to be included because box props already include it.

Reply:
Didn't know that!

Reply:
Rather – the box _types_ include it. Personally don't think we need to include these sorts of pass-through props in PropTypes.